modules/end-user-guide/pages/optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc

```diff
@@ -0,0 +1,110 @@
+
+== URL parameter concatenation
+
+The URL for starting a new workspace supports concatenation of multiple URL parameters by using `&` with the following URL syntax:
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?__<url_parameter_1>__&__<url_parameter_2>__&__<url_parameter_3>__
+----
+
+== URL parameter for the workspace IDE
+
+Che-Theia is currently the default integrated development environment (IDE) that a workspace loads with if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.
+
+The URL parameter for specifying another supported IDE is `che-editor=__<editor_key>__`.
+
+The URL for starting a new workspace has the following syntax with the `che-editor=__<editor_key>__` parameter:
+
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?che-editor=__<editor_key>__
+----
+
+.The `__<editor_key>__` values for the supported IDEs
+|===
+| IDE | `__<editor_key>__` value | Note
+
+ifeval::["{project-context}" == "che"]
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+| `eclipse/che-theia/latest`
+| When using the `stable` link:https://github.com/che-incubator/chectl/[chectl] channel, this IDE loads as a default in a new workspace without this URL parameter.
+
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+| `eclipse/che-theia/next`
+| When using the `next` link:https://github.com/che-incubator/chectl/[chectl] channel, this IDE loads as a default in a new workspace without this URL parameter.
+endif::[]
+
+ifeval::["{project-context}" == "crw"]
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+|`eclipse/che-theia/latest`
+| This is the default IDE that loads in a new workspace without this URL parameter.
+endif::[]
+
+| link:https://github.com/che-incubator/che-code[Che-Code]
+| `che-incubator/che-code/insiders`
+|
+
+| link:https://www.jetbrains.com/help/idea/discover-intellij-idea.html[IntelliJ IDEA Community Edition]
+| `che-incubator/intellij-community/latest`
+|
+
+| link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
+| `che-incubator/pycharm/latest`
+|
+
+|===
+
+== URL parameter for starting duplicate workspaces
+
+Visiting a URL for starting a new workspace results in a workspace that is based on a combination of a devfile and a linked Git repository.
+
+In some situations, you may need to have multiple workspaces that are duplicates in terms of the devfile and the linked Git repository. You can do this by visiting the same URL for starting a new workspace with a URL parameter. The URL parameter for starting a duplicate workspace is `new`.
+
+The URL for starting a new workspace has the following syntax with the `new` parameter:
```

Comment:
```suggestion
```
Is this sentence redundant with the one at the end of the previous paragraph? Maybe keep one and delete the other? When I see both, I start rereading  and comparing them repeatedly to figure out what the differences might be.

Reply:
The first sentence says what the parameter is, and the second sentence shows how the parameter is used in the URL. I have both sentence for each parameter, as it seems to me that I can't drop either without missing out basic information. I could merge these two sentences into one, but that would result in a long sentence with the same meaning.

Reply:
Let have a second look. I'll use the first sentence with the block of the second sentence.

Reply:
![image](https://user-images.githubusercontent.com/78564304/162475704-de65d6fb-a42e-4480-9331-3e969c3728e5.png)